CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Added
 
+## [23.2.0] - 2020-06-29
+
+### Added in 23.2.0
+
+- Admin can create empty classes
```

Comment:
Please have a look at the specification of the changelog. You should only add your changes to the `[Unreleased]` section (which is later renamed as part of the release process).